SUMMARY.md

```diff
@@ -32,6 +32,7 @@
   * [Custom middleware](examples/custom-middleware/README.md)
     * [Enriching the metadata](examples/custom-middleware/enriching-the-metadata.md)
     * [Reject machine-to-machine traffic using custom metadata ACL rules](examples/custom-middleware/reject-machine-to-machine-traffic-using-custom-metadata-ACL-rules.md)
+    * [Tenant group manipulation](examples/custom-middleware/tenant-group-manipulation.md)
```

Comment:
I'd call this task "Reordering available tenancies"

Reply:
Done ✔️